README.md

```diff
@@ -48,6 +48,7 @@ workdir: <Directory to use for creating OCP configs>
 storage_type: <Storage type used in the cluster. Eg: nfs (Note: Currently NFS provisioner is not configured using this playbook. This variable is only used for setting up image registry to EmptyDir if storage_type is not nfs)>
 log_level: <Option --log-level in openshift-install commands. Default is 'info'>
 release_image_override: '<This is set to OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE while creating ign files. If you are using internal artifactory then ensure that you have added auth key to the pull_secret>'
+rhcos_pre_kernel_options: <List of day-1 kernel options for RHCOS nodes eg: ["rd.multipath=default","root=/dev/disk/by-label/dm-mpath-root"]>
```

Comment:
The PR looks good, but I'm wondering if we can come up with a better var name ?

Reply:
eg. rhcos_day1_kernel_options ?

Reply:
thoughts @yussufsh @Prajyot-Parab  ?

Reply:
I feel we should add everything from rhcos_kernel_options in the manifest. No need of a new variable.

Reply:
As per my understanding, we can use the same existing var but then we would need logic to decide when to use it either pre or post installation. If that adds additional var then we can stick to current PR changes itself.

IMO we can stick to current approach with renaming the vars as it would be much simpler and easier to understand.

^^^ @aishwaryabk @pravin-dsilva